packages/preferences/tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig.json",
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+		"types": [ "gutenberg-env" ],
+		"checkJs": false
+	},
+	"include": [ "src/**/*" ],
```

Slight preference to use `src` which should get the same result but the glob isn't necessary. TS should handling finding the correct files.

```suggestion
	"include": [ "src" ],
```

I need to pick up parts of https://github.com/WordPress/gutenberg/pull/63605 which will remove some of the boilerplate in these package tsconfig files.

---

`"include": [ "src/**/*" ]` is used everywhere else in Gutenberg. If it should be changed, then ideally it's changed holistically for all packages.